thoth/package_analyzer/cli.py

```diff
@@ -123,15 +124,18 @@ def python(
     dry_run: bool = False,
 ):
     """Fetch digests for packages in Python ecosystem."""
+    start_time = time.time()
```

Comment:
Please use [time.monotonic()](https://docs.python.org/3/library/time.html#time.monotonic) for measuring this.

Reply:
Thanks @fridex !